easybuild/easyconfigs/s/SIP/SIP-4.18.1-foss-2015a-Python-3.5.1.eb

```diff
@@ -0,0 +1,34 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Bart Verleye
+# Center for eResearch, Auckland
+easyblock = 'ConfigureMakePythonPackage'
+
+name = 'SIP'
+version = '4.18.1'
+
+homepage = 'http://www.riverbankcomputing.com/software/sip/'
+description = """SIP is a tool that makes it very easy to create Python bindings for C and C++ libraries."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls=['http://downloads.sourceforge.net/project/pyqt/sip/sip-%(version)s/']
+
+python = 'Python'
+pyver = '3.5.1'
+versionsuffix = '-%s-%s' % (python, pyver)
```

Comment:
Please update this to the new style, and move `versionsuffix` to immediately below `version`.